cluster/common.sh

```diff
@@ -446,8 +446,19 @@ function yaml-quote {
 # Builds the RUNTIME_CONFIG var from other feature enable options (such as
 # features in alpha)
 function build-runtime-config() {
-  # There is nothing to do here for now. Just using this function as a placeholder.
-  :
+  # If a policy provider is specified, enable NetworkPolicy API.
+  if [[ -n "${NETWORK_POLICY_PROVIDER}" ]]; then
```

Comment:
this broke all providers except GCE, since `NETWORK_POLICY_PROVIDER` isn't set anywhere else. this should probably instead be `${NETWORK_POLICY_PROVIDER:-}`